@here/harp-mapview/test/VisibleTileSetTest.ts

```diff
@@ -51,11 +52,10 @@ class Fixture {
             MapViewDefaults.extendedFrustumCulling,
             params.tileWrappingEnabled
         );
-        this.vts = new VisibleTileSet(
-            this.frustumIntersection,
-            this.tileGeometryManager,
-            MapViewDefaults
-        );
+        this.vts = new VisibleTileSet(this.frustumIntersection, this.tileGeometryManager, {
+            ...MapViewDefaults,
+            clipPlanesEvaluator: new TiltViewClipPlanesEvaluator()
```

Comment:
Personally I would replace this with simple factory method (createDefaultClipPlanesEvaluator()), of course we all know the reason that default (instance) can not be used, but factory gives us flexibility of changing it once and everywhere if needed.